source/Calamari/Kubernetes/Authentication/GoogleKubernetesEngineAuth.cs

```diff
@@ -31,7 +31,7 @@ public GoogleKubernetesEngineAuth(GCloud gcloudCli,GkeGcloudAuthPlugin authPlugi
             this.log = log;
         }
 
-        public bool TryConfigure(bool useVmServiceAccount, string @namespace)
+        public bool TryConfigure(string @namespace)
```

Comment:
I _really_ want to get rid of the `bool Try...` semantics in a later commit. returning true or false everywhere rather than just throwing an exception feels unecessary when it typically is the result of some exceptional circumstances that we want to just stop continued execution on